lua/apg/cl_menu.lua

```diff
@@ -327,7 +327,7 @@ properties.Add( "apgoptions", {
 
     Filter = function( self, ent, ply ) -- A function that determines whether an entity is valid for this property
         if not ply:IsSuperAdmin() then return false end
-        return (ent.GetClass and ent:GetClass() and IsValid(ent) and ent:EntIndex() > 0)
+        return ent.GetClass and ent:GetClass() and IsValid(ent) and ent:EntIndex() > 0

```

Comment:
Might make more sense to turn these into multiple guards:

```suggestion
if not ent:GetClass() then return false end
if not IsValid(ent) then return false end
if not ent:EntIndex() > 0 then return false end

return true
```

Allows for easy manipulation in the future, separates the checks, reduces line length, and IMO is easier on the eyes :)